IDEO.md

```diff
@@ -0,0 +1,34 @@
+<img src="./etc/image/OKG_Badge_600x600.png" width="200" align="right"/>
+
+# Integrated Development Environment for Ontologies
+
+# Table of contents
+
+* [Use case development](#use-case-development)
+
+
+# Introduction
+
+EDMC OKG presents the Integrated Development Environment for Ontologies (IDEO)... It is based on and is a generalizaton of FIBO BTDM methodology descibed in [FIBO_BTDM.docx](./etc/site/fibo/doc/FIBO_BTDM.docx)
+
+IDEO for modular ontologies...
+
+.. move through a rigorous process and become available to the industry.  
+
+
+
+
+## IDEO Process
+This section discusses the IDEO based on GitHub, Ontology Publisher, and Jenkins being its core components.
```

Comment:
The point of an IDE is to provide an integrated user experience, including editing. We don't have that so use of "IDE" is not appropriate.

Reply:
I agree with @rivettp  here.  But that name IDEO is used throughout.  What should we call it instead?    Maybe a Continuous Integraion Process for Ontologies (CIPO)?  That seems correct; this is continuous integration, and it is a process, and it is tuned for ontologies.